### ISSUE TITLE
Update JBump fork

### ISSUE THREAD

**Issue by SonicGDX** [libgdx/gdx-liftoff#168]
The [fork](https://github.com/tommyettinger/jbump) of jBump which seems to be used in gdx-liftoff (and is mentioned in awesome libGDX) appears to be out of date compared to the [upstream](https://github.com/implicit-invocation/jbump/) - fork is on on version 1.0.1 compared to 1.0.2 on the upstream (and upstream has a few additional commits after)

**Comment by tommyettinger** [comment]
Yeah, the history there is complicated. I had forked initially just because there was a file, `Aux.java`, which made cloning the repo impossible on Windows. Then I made some possibly-ill-advised changes, though they may have been good... it has been a while. Then I was made a contributor to the original repo and brought my changes back there. Then someone, maybe me, released 1.0.2... That may have just been to establish the higher version at the new URL. I think 1.0.2 is similar to 1.0.1, but I can investigate the actual differences.
